app/dashboard/page.tsx

```diff
@@ -1,12 +1,42 @@
+import Card from '@/ui/Card';
+import CardTable from '@/ui/CardTable';
+
 export default function Page() {
   return (
     <div className="flex h-screen w-full">
-      <div className="m-auto">
-        <h1 className="text-center text-4xl font-bold">
-          Amuse toi avec <span className="rounded-md bg-white p-1 text-black ">Next.js</span> et{' '}
-          <span className="text-blue-500">Tailwindcss</span> !
-        </h1>
+      <div className="">
+        <h1 className="text-2xl font-bold tracking-tight">Statistique</h1>
+        <div className="flex flex-col space-x-0 md:flex-row md:space-x-9 pt-5">
+          <Card></Card>
+          <Card></Card>
+          <Card></Card>
```

Comment:
Puisque ce sont des composants, ce sont d'office des balises autos-fermantes (il n'y a pas de props children par exemple)

```suggestion
          <Card />
          <Card />
          <Card />
```

Reply:
Merci pour la précision